cmd/log-parser/README.md

```diff
@@ -92,7 +92,9 @@ To merge all logs:
    ```
    $ sudo chown $USER *.log
    ```
-1. Run the script:
+1. Install and run the script:
```

Comment:
This was originally my mistake - `kata-log-parser` is not a script, so could you change this to:

> The run the program:

Could you make the install part a separate step before the "run" part?